packages/styles/src/core/layout/sidebar/README.md

```diff
@@ -1,103 +1,137 @@
+    
+    <div class="as-map-wrapper" style="background: #B5E0F9;">
+      <!-- Put your map here -->
+    </div>
+  </main>
+</div>
 ```
 
-```code
-lang: html
----
-<body class="as-app">
-  <div role="tablist" class="as-toolbar-tabs as-tabs">
-    <button onclick="showMap(event)" role="tab" class="as-tabs__item as-tabs__item--active">MAP</button>
-    <button onclick="showLeft(event)" role="tab" class="as-tabs__item">LEFT</button>
-    <button onclick="showRight(event)" role="tab" class="as-tabs__item">RIGHT</button>
-  </div>
+### .as-sidebar--right
 
+Puts the sidebar on the right of the map.
+
+```html
+<div class="as-app"  style="height: 300px; width:100%;">
   <main class="as-app-content">
-    <aside class="as-sidebar as-sidebar--left as-sidebar--xl">
-      <h1>Left Sidebar (XL)</h1>
-      <p>
-        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nesciunt aliquam nihil nulla quo, cum excepturi commodi saepe dicta
-        impedit accusamus, nisi adipisci repellendus expedita rem dolores suscipit! Cumque, nulla culpa?
-      </p>
-    </aside>
+    <div class="as-map-wrapper" style="background: #B5E0F9;">
+      <!-- Put your map here -->
+    </div>
 
-    <div class="as-map-wrapper">
-      <div id="map"></div>
+    <aside class="as-sidebar as-sidebar--right" style="background: #80B622;"></aside>
+  </main>
+</div>
+```
+
+
+
+### .as-sidebar--l
+
+Makes the sidebar to be 360px width.
+
+```html
+<div class="as-app"  style="height: 300px; width:100%;">
+  <main class="as-app-content">
+
+    <aside class="as-sidebar as-sidebar--left as-sidebar--l" style="background: #80B622;"></aside>
+    
+    <div class="as-map-wrapper" style="background: #B5E0F9;">
+      <!-- Put your map here -->
     </div>
+  </main>
+</div>
+```
+
+### .as-sidebar--xl
+
+Makes the sidebar to be 460px width.
+
+```html
+<div class="as-app"  style="height: 300px; width:100%;">
+  <main class="as-app-content">
 
-    <aside class="as-sidebar as-sidebar--right">
-      <h1>Right Sidebar</h1>
-      <p>
-        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nesciunt aliquam nihil nulla quo, cum excepturi commodi saepe dicta
-        impedit accusamus, nisi adipisci repellendus expedita rem dolores suscipit! Cumque, nulla culpa?
-      </p>
-    </aside>
+    <aside class="as-sidebar as-sidebar--left as-sidebar--xl" style="background: #80B622;"></aside>
+    
+    <div class="as-map-wrapper" style="background: #B5E0F9;">
+      <!-- Put your map here -->
+    </div>
   </main>
-</body>
+</div>
 ```
 
-## Mobile Sidebar
-In order to make a sidebar visible on a mobile phone or any device with a small screen you need to add the `--visible` class modifier you can toggle this class from wherever you want but we recommend using **toolbar-tabs**.
+
+### .as-sidebar--visible
+
+Makes the sidebar visible on small screens occupying all the space available in the`.app-content` and covering the map.
```

Comment:
Makes the sidebar visible on small screens filling all the space available in the`.app-content` and covering the map.

#OccupyWallStreet